scheduler-deploy

```diff
@@ -75,6 +75,33 @@ fn-set-resource-constraints() {
   fi
 }
 
+fn-set-healthchecks() {
+  declare APP="$1" PROC_TYPE="$2" DEPLOYMENT_FILE="$3"
+  local TMP_DIR=$(mktemp -d "/tmp/${FUNCNAME[0]}.XXXX")
+  local APP_JSON_TMPL="$TMP_DIR/app.json.sigil"
+  local APP_JSON_FILE="$TMP_DIR/app.json"
+  local PROBE_FILE="$TMP_DIR/probe.json"
+  local DEPLOYMENT_TMP_FILE="$TMP_DIR/deployment.json"
+  trap 'rm -rf "$TMP_DIR" > /dev/null' RETURN INT TERM EXIT
+
+  copy_from_image "$IMAGE" "app.json" "$APP_JSON_TMPL" 2>/dev/null || true
+
+  if [[ ! -f "$APP_JSON_TMPL" ]]; then
+    return 0
+  fi
+
+  SIGIL_PARAMS=(APP="$APP" IMAGE="$IMAGE" PROCESS_COUNT="$PROC_COUNT" PROCESS_TYPE="$PROC_TYPE" PORT="5000")
+  sigil -f "$APP_JSON_TMPL" "${SIGIL_PARAMS[@]}" | cat -s >$APP_JSON_FILE
```

Comment:
It's not clear what the templating is for. Can you add some documentation to this?

Reply:
Also an explanation in a PR comment would be good, its really not clear what's going on here or what the intended usage is...

Reply:
Thanks for the feedback I've removed the unnecessary template parameters, added a comment in the code, and added docs and example to README.md. Please review again.